draft-ietf-dprive-dnsoquic.md

```diff
@@ -207,10 +207,11 @@ initiated transactions within established DoQ connections.
 That is, only the initiator of the DoQ connection may send queries over the
 connection.
 
-DSO supports server-initiated transactions within existing connections, however
-DSO is not applicable to DNS over HTTP since HTTP has its own mechanism for
-managing sessions, and this is incompatible with the DSO; the same is true for
-DoQ.
+DSO does support server-initiated transactions within existing connections.
+However DoQ as defined here does not meet the criteria for an applicable
+transport for DSO because it does not guarantee in-order delivery of messages,
+see Section 4.2 of {{?RFC8490}}.
+
```

Comment:
Interesting point. DoQ does indeed send and process multiple queries in parallel. I can see how DSO would send series of "Example.com IN A is not 1.2.3.4", "strike that, 1.2.3.5", etc. If that was processed in a different order, that would be bad. I suppose we could fix that -- either some explicit number of DSO announcements, or some server behavior, or all announces triggered by a "subscribe" being forced on a  single stream. How about adding "support of DSO in DoQ would require an extension or a revision of this specification"?